addons/edi_notifier/models/email_notifier.py

```diff
@@ -35,16 +39,24 @@ def _notify(self, notifier, event_type, recs):
                 template = template.with_context(attachments=attachments)
             email_values = None
             if attachments:
-                email_values = {'attachment_ids': attachments.mapped('id')}
+                email_values = {"attachment_ids": attachments.mapped("id")}
             template.send_mail(rec.id, force_send=True, email_values=email_values)
 
+    def check_disbaled_edi_notifier(self):
+        return config.get_misc("edi", "disable_edi_notifications", False)
```

Comment:
For the other safety catches (edi, print) there is a setting where the safety catch name can be changed. Would be good to follow that approach instead of hardcoding the safety catch name.